lazy_dataset/database.py

```diff
@@ -229,10 +230,20 @@ def __init__(self, json_path: [str, Path]):
     @property
     def data(self):
         if self._data is None:
-            path = Path(self._json_path).expanduser()
+            paths = [Path(p).expanduser() for p in self._json_path]
 
-            with path.open() as fd:
+            with paths[0].open() as fd:
                 self._data = json.load(fd)
+
+            for path in paths[1:]:
+                with path.open() as fd:
+                    dataset = json.load(fd)['datasets']
```

Comment:
Can you also merge alias? And add an assert, that at the moment the second, third, ... json are only allowed to have the keys `datasets` and `alias`?